doc/examples/bundle_extraction.py

```diff
@@ -153,70 +207,229 @@
 
-sft_cst_l = load_trk(model_cst_l_file, "same", bbox_valid_check=False)
-model_cst_l = sft_cst_l.streamlines
+"""
 
-recognized_cst_l, cst_l_labels = rb.recognize(model_bundle=model_cst_l,
-                                              model_clust_thr=5.,
-                                              reduction_thr=10,
-                                              reduction_distance='mam',
-                                              slr=True,
-                                              slr_metric='asymmetric',
-                                              pruning_distance='mam')
+Now, let's increase the reduction_thr and pruning_thr values further.
 
 """
-let's visualize extracted Corticospinal Tract (CST) Left bundle and model
-bundle together
+
+rb = RecoBundles(moved, verbose=True, rng=np.random.RandomState(2001))
+
+recognized_af_l, af_l_labels = rb.recognize(model_bundle=model_af_l,
+                                            model_clust_thr=0.1,
+                                            reduction_thr=25,
+                                            pruning_thr=12,
+                                            reduction_distance='mdf',
+                                            pruning_distance='mdf',
+                                            slr=True)
+
+"""
+let's visualize extracted Arcuate Fasciculus Left bundle.
 """
 
 interactive = False
 
 scene = window.Scene()
 scene.SetBackground(1, 1, 1)
-scene.add(actor.line(model_cst_l, colors=(.1, .7, .26)))
-scene.add(actor.line(recognized_cst_l, colors=(.1, .1, 6)))
+scene.add(actor.line(recognized_af_l))
 scene.set_camera(focal_point=(-18.17281532, -19.55606842, 6.92485857),
-                 position=(-360.11, -340.46, -40.44),
+                 position=(-360.11, -30.46, -40.44),
                  view_up=(-0.03, 0.028, 0.89))
-window.record(scene, out_path='CST_L_recognized_bundle.png',
+window.record(scene, out_path='AF_L_recognized_bundle3.png',
               size=(600, 600))
 if interactive:
     window.show(scene)
 
+"""
+.. figure:: AF_L_recognized_bundle3.png
+   :align: center
+
+   Extracted Arcuate Fasciculus Left bundle
+
+"""
 
 """
-.. figure:: CST_L_recognized_bundle.png
+
+Save the bundle as a trk file. Let's save the recognized bundle in the
+common space (atlas space), in this case, MNI space.
+
+"""
+
+reco_af_l = StatefulTractogram(recognized_af_l, atlas_header,
+                               Space.RASMM)
+save_trk(reco_af_l, "AF_L_rec_3.trk", bbox_valid_check=False)
+
+"""
+
+Let's save the recognized bundle in the original space of the subject anatomy.
+
+"""
+
+reco_af_l = StatefulTractogram(target[af_l_labels], target_header,
+                               Space.RASMM)
+save_trk(reco_af_l, "AF_L_org_3.trk", bbox_valid_check=False)
+
+
+"""
+Let's apply auto-calibrated RecoBundles on the output of standard RecoBundles.
+This step will filter out the fuzzy streamlines. This time, the RecoBundles'
```

Comment:
```suggestion
This step will filter out the fuzzy streamlines. This time, the RecoBundles'
```
What does "fuzzy" mean here? It looks like maybe it filters outliers. Is that what is meant here?